main.py

```diff
@@ -1,10 +1,46 @@
-import init_django_orm  # noqa: F401
+import json
+
+import init_django_orm
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
```

Comment:
The context manager is overloaded here. According to the checklist, you should only use the context manager block for reading the file. Consider moving the logic that processes the data outside of the `with` block to improve code readability and maintainability .